openshift/system/Makefile

```diff
@@ -1,12 +1,6 @@
-MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
-PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
-PROJECT = $(subst @,,$(notdir $(subst /workspace,,$(PROJECT_PATH))))
-
 NAME=system-os
 NAMESPACE=quay.io/3scale
 SYSTEM_VERSION ?= 0.0.5
-RAILS_ENV=production
-PORT=3000
 VERSION := $(SYSTEM_VERSION)
 LOCAL_IMAGE := $(NAME):$(VERSION)
 REMOTE_IMAGE := $(NAMESPACE)/$(LOCAL_IMAGE)
```

Comment:
I think these `LOCAL_IMAGE` and `REMOTE_IMAGE` could be improved BTW. I personally override `SYSTEM_VERSION` with `git rev-parse --abbrev-ref HEAD` (instead its default "0.0.5" that no longer means anything) and I use "porta" instead of "system-os" for the name of the image.

But let's leave this suggestion for another PR 😉

Reply:
Sure.